python/main.py

```diff
@@ -1,42 +1,95 @@
 import os
 import logging
 import pathlib
+import sqlite3
 from fastapi import FastAPI, Form, HTTPException
 from fastapi.responses import FileResponse
 from fastapi.middleware.cors import CORSMiddleware
 
+import sqlite3
+import json
+
 app = FastAPI()
 logger = logging.getLogger("uvicorn")
 logger.level = logging.INFO
 images = pathlib.Path(__file__).parent.resolve() / "images"
-origins = [ os.environ.get('FRONT_URL', 'http://localhost:3000') ]
+origins = [os.environ.get('FRONT_URL', 'http://localhost:3000')]
 app.add_middleware(
     CORSMiddleware,
     allow_origins=origins,
     allow_credentials=False,
-    allow_methods=["GET","POST","PUT","DELETE"],
+    allow_methods=["GET", "POST", "PUT", "DELETE"],
     allow_headers=["*"],
 )
 
+
+# filename = "items.json"
+DatabaseName = "../db/mercari.sqlite3"
+
+
+# def format_items(items):
+#     items_format = []
+#     for item in items:
+#         item_format = {"name": item[0], "category": item[1]}
+#         items_format.append(item_format)
+
+#     return {"items": f"{items_format}"}
+
+
 @app.get("/")
 def root():
     return {"message": "Hello, world!"}
 
+
 @app.post("/items")
-def add_item(name: str = Form(...)):
+def add_item(name: str = Form(...), category: str = Form(...)):
+
+    # connect
+    conn = sqlite3.connect(DatabaseName)
+    # cursor
+    cur = conn.cursor()
+
+    cur.execute("INSERT INTO items VALUES (?,?)",
```

Comment:
You need to create an item table in database.
Only then you will be able to insert or select!

If you are using VSCode, an extension called [SQLite Viewer](https://marketplace.visualstudio.com/items?itemName=qwtel.sqlite-viewer) might help you to visualize what's going on inside the sql database.